pennylane/measurements/probs.py

```diff
@@ -299,23 +300,24 @@ def marginal_prob(self, prob, wire_order, batch_size):
         # reshape the probability so that each axis corresponds to a wire
```

Comment:
Can you remind me, why do we have `marginal_probs` in the device and in the measurement? Which one is used when?

Reply:
the MeasurementProcess `process_state` (and subsequently, `marginal_prob`) don't seem to be used at all today. It's effectively a copy of code, in anticipation for the device-specific code to be removed in the future. Today, the device entrypoint is [in staticstics()](https://github.com/PennyLaneAI/pennylane/blob/1774fe36f31f7de6266cf44983c85dfb561ec0fc/pennylane/_qubit_device.py#L818), which eventually finds its way to `device.marginal_prob`. In the future, we won't call `self.probability`. Instead, we'll call `probs_MP.process_state()`. But for today, I need to update both.

Reply:
Thanks, all good then :+1: